internal/log/log.go

```diff
@@ -15,170 +14,113 @@ var (
 
 // SetLevel sets minimum level for logs, logs with level above specified value will not be printed.
 func SetLevel(levelName string) {
-	level, ok := LevelsByName[levelName]
+	level, ok := levelsByName[levelName]
```

Comment:
Is this function correct? What happens if there is no `levelName` in the map? Shouldn't L23 set Error level to `DeafultLevel`? (`level` will be an empty string if `ok` is false)